ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/monitor-layer1.j2

```diff
@@ -11,6 +11,7 @@
 | Layer 1 Event | Logging |
 | ------------- | ------- |
 | MAC fault | {{ monitor_layer1.logging_mac_fault | arista.avd.default(false) }} |
+| Logging Transceiver | {{ monitor_layer1.logging_transceiver.enabled | arista.avd.default(false) }} |
```

Comment:
Please enclose each line in the table like this. It will confuse to get `false` for the platforms not supporting the individual settings.
```suggestion
{%     if monitor_layer1.logging_transceiver.enabled is arista.avd.defined %}
| Logging Transceiver | {{ monitor_layer1.logging_transceiver.enabled }} |
{%     endif %}
```

Reply:
Fixed.